### DISCUSSION TITLE
Section 2.3, Exercise 7: Why are we using the column nox to create the quadratic term?

### DISCUSSION THREAD

**Discussion by eraunio** [pykale/transparentML#53] [Q&A] [answered]
This exercise is about comparing a linear and quadratic fit by regressing ptratio onto crim. However, why does the solution suggest using the cube of nox for creating the quadratic term (rather than a squaring up the ptratio)?

**Comment by Mdnaimulislam** [comment, marked as answer]
Thank you very much for bringing this to our attention. You are absolutely correct in your observation, and I appreciate your diligence in identifying this error. The use of the cube of "nox" in our initial solution was indeed a mistake. For the purpose of comparing linear and quadratic fits by regressing "ptratio" onto "crim", the correct approach is to square the "ptratio" variable to create the quadratic term.

The error in the solution was an oversight, and I have since corrected the materials on the website to accurately reflect the proper approach. Discussions and feedback like this are invaluable, as they help ensure the accuracy and reliability of our educational content. I am grateful for your engagement and your contributions to a productive learning environment. If you wish, you can view the changes in PR #54.

Please continue to share your insights and questions as they arise.
